authgear/templates/authgear-admin-server.yaml

```diff
@@ -170,6 +170,27 @@ spec:
   externalName: {{ include "authgear.nameAdmin" . }}.{{ .Release.Namespace }}.svc.cluster.local
   ports:
   - port: 80
+{{- if .Values.authgear.adminServer.pdb.enabled}}
```

Comment:
Is `pdb` a well known shorthand of PodDisruptionBudget?

Reply:
It is used in Bitnami, for example https://github.com/bitnami/charts/blob/main/bitnami/redis/values.yaml#L666

In ingress-nginx, `pdb` is not used https://github.com/kubernetes/ingress-nginx/blob/main/charts/ingress-nginx/values.yaml#L385 I think this is not a good design because I prefer  having `pdb.enabled = false` to turn off PodDisruptionBudget.

Reply:
Also you can run `kubectl get pdb` to get PodDisruptionBudget, so I think it is known to people who know what is PodDisruptionBudget.